pf9/modules/analytics_utils.py

```diff
@@ -15,6 +15,19 @@
 # Project 'TEST - Python-testing-V2' in segment
 segment_dev_write_key = 't1tdSZocQ49Tx3G66lUjJXkOXjUSYRKx'
 
+
+def strip_account_url(account_url):
+    """
+        Remove http/https for an URL
+    """
+    if account_url:
+        if 'https://' in account_url:
```

Comment:
https://docs.python.org/3/library/urllib.parse.html - maybe easier with this.

Reply:
Even with urllib, I found that there is no easy way to strip the scheme. So I might just stick to my current changes.